policies/coding-style.md

```diff
@@ -0,0 +1,624 @@
+# OpenSSL coding style
+
+This document describes the coding style for the OpenSSL project. It is
+derived from the [Linux kernel coding style][1].
+
+This guide is not distributed as part of OpenSSL itself. Since it is
+derived from the Linux Kernel Coding Style, it is distributed under the
+terms of the [kernel license][2], available here:
```

Comment:
"available here"....errr where exactly?

Reply:
Oops.  There had to be at least one.
Fixed now.